game/src/graphic/textures/TextureHandler.java

```diff
@@ -15,17 +18,44 @@
 /**
  * A texture handler class for managing textures paths for further use.
  *
- * <p>Singleton (possibly thread-safe).
+ * <p>Singleton.
```

Comment:
??? was sagt mir diese zeile/info? 

entweder raus damit oder eine vernünftige erklärung, warum das ein singleton ist. 

(**dass** es ein singleton ist, sehe ich selbst. **warum** wäre die info, die ich brauche.)

Reply:
Es ist ein Singleton, damit alle Pfade nur einmal am Anfang gesucht werden.

Aber mMn kann die Zeile weg.

Reply:
diese erklärung fehlt in der javadoc.